### PR TITLE
Add prettier-plugin-astro to list of plugins

### PR DESCRIPTION
**Pull request by jasikpark**
## Description

This adds https://github.com/withastro/prettier-plugin-astro to the list of prettier plugins

I added `@withastro` as the creator, but maybe it makes more sense to link to https://github.com/withastro/prettier-plugin-astro/graphs/contributors as the creators?

## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: docs/plugins.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ To turn off plugin autoloading, use `--no-plugin-search` when using Prettier CLI
 
 ## Community Plugins
 
+- [`prettier-plugin-astro`](https://github.com/withastro/prettier-plugin-astro) by [**@withastro**](https://github.com/withastro)
```
Comment (fisker) [inline docs/plugins.md:60]: The plugin names are alphabetical sorted, you might want move to the second one.

Comment (jasikpark) [reply docs/plugins.md]: done, thx

**Comment by jasikpark** [comment]
> > but maybe it makes more sense to link to [withastro/prettier-plugin-astro/graphs/contributors](https://github.com/withastro/prettier-plugin-astro/graphs/contributors) as the creators?
> 
> You can, if you like.

Updated

**Comment by fisker** [comment]
You can add words to https://github.com/prettier/prettier/blob/main/cspell.json#L4

**Comment by jasikpark** [comment]
> You can add words to [https://github.com/prettier/prettier/blob/main/cspell.json#L4](https://github.com/prettier/prettier/blob/main/cspell.json?rgh-link-date=2022-08-10T14%3A22%3A38Z#L4)

Done, thanks

**Comment by jasikpark** [comment]
Sorry for not checking previously 😅 didn't think that a markdown change might result in CI failing, though that makes sense - i forgot about y'all's cspell lint step

**Review by fisker — APPROVED** [review]

**Review by sosukesuzuki — APPROVED** [review]
Thanks, I love Astro :heart:

**Pull request merged**